src/diffusion_for_multi_scale_molecular_dynamics/models/diffusion_mace.py

```diff
@@ -166,7 +189,7 @@ def __init__(
         # define the "0e" representation as a constant to avoid "magic numbers" below.
         scalar_irrep = o3.Irrep(0, 1)
 
-        # Apply an MLP with a bias on the scalar diffusion time-like  input.
+        # Apply an MLP with a bias on the scalar diffusion time-like input and 1-hot atom type
```

Comment:
This comment tripped me up initially. I think it would be useful to modify it to say:
`An MLP will be used to mix the diffusion time-like input (the 'diffusion scalar', a global quantity) and the 1-hot atom types (the 'node scalars')